content/docs/_index.md

```diff
@@ -8,7 +8,6 @@ bookCollapseSection: true
 
 * [About]({{< ref "about" >}}) - Key principles of Packit.
 * [Onboarding Guide]({{< ref "guide" >}}) - How to start using Packit.
-* [Packit Service]({{< ref "packit-service" >}}) - GitHub App that helps you continuously ensure that your project work in Fedora OS.
```

Comment:
This link is still valid, is it not? So it should probably be kept here.

Reply:
But it's a duplicate of the one above. (There are no longer two pages.)